alf/algorithms/ppo_loss.py

```diff
@@ -98,58 +99,19 @@ def __init__(self,
         self._check_numerics = check_numerics
 
     def _pg_loss(self, training_info: TrainingInfo, advantages):
-        current_policy_distribution = training_info.action_distribution
-
-        sample_action_log_probs = tfa_common.log_probability(
-            training_info.collect_action_distribution, training_info.action,
-            self._action_spec)
-        sample_action_log_probs = tf.stop_gradient(sample_action_log_probs)
-
-        action_log_prob = tfa_common.log_probability(
-            current_policy_distribution, training_info.action,
-            self._action_spec)
-        if self._log_prob_clipping > 0.0:
-            action_log_prob = tf.clip_by_value(action_log_prob,
-                                               -self._log_prob_clipping,
-                                               self._log_prob_clipping)
-        if self._check_numerics:
-            action_log_prob = tf.debugging.check_numerics(
-                action_log_prob, 'action_log_prob')
-
-        # Prepare both clipped and unclipped importance ratios.
-        importance_ratio = tf.exp(action_log_prob - sample_action_log_probs)
-        if self._check_numerics:
-            importance_ratio = tf.debugging.check_numerics(
-                importance_ratio, 'importance_ratio')
-        importance_ratio_clipped = tf.clip_by_value(
-            importance_ratio, 1 - self._importance_ratio_clipping,
-            1 + self._importance_ratio_clipping)
-
+        name_scope = 'PPOLoss'
+        importance_ratio, importance_ratio_clipped = value_ops.action_importance_ratio(
```

Comment:
it's better to user key=value form to call the function, it's more robust to future changing of the function signature.

Reply:
done.